README.md

```diff
@@ -55,3 +55,18 @@ If a single output file is provided, **hwgraph** plots for each benchmark :
 If multiple output files are passed as arguments, and only if they were generated with the same script file, **hwgraph** will compare for each benchmark the performance metrics.
 
 For more details, see the specific documentation.
+
+# Requirements
+## Mandatory
+- python >= 3.9
```

Comment:
Don't we want to document the runtime python dependencies? Or maybe link to requirements/base.in ?

Reply:
I asked myself the same question.
Maybe linking is better

Reply:
Let's add the link in another commit/PR.

Reply:
Why in another PR ?

Reply:
Just in case. Works with this PR.